content/components/action-bar.mdx

```diff
@@ -74,7 +105,16 @@ Dividers can be added to visually group related buttons.
 
 ### Overflow menu
 
-When the buttons don't fit in the available space, an overflow button ("kebab" icon) is added at the end of the action bar signaling that there are more actions available. Clicking on the overflow button opens a menu with the remaining actions that didn't fit.
+{/* <!-- TODO: determine how/if we want to support hiding items out of order --> */}
+
+When actions are put in the overflow menu, an icon button (["kebab" icon](/foundations/icons/kebab-horizontal-16)) is added at the end of the action bar signaling that there are more actions available. Clicking on the overflow button opens a menu with the remaining actions that didn't fit.
+
+Dividers from the action bar are preserved in the overflow menu.
+
+There are two ways that actions can be pushed into the overflow menu:
+
+- When the buttons don't fit in the available horizontal space
+- By explicitly defining actions that will only appear in the overflow menu
```

Comment:
Delete this if we choose not to let consumers force actions to only appear in the overflow menu.

Reply:
Yeah, lets implement this only if we find a usecase.